src/profile/Account.vue

```diff
@@ -0,0 +1,127 @@
+              <v-icon>mdi-pencil</v-icon>
+            </v-list-item-avatar>
+
+            <v-switch :loading="toggling" v-if="item.hasSwitch" v-model="hidden" @change="toggle()" />
+          </v-list-item>
+        </template>
+      </v-card>
+    </v-list>
+
+    <v-alert :value="hasUnverifiedEmails" type="error">
+      <span>{{ $vuetify.lang.t('$vuetify.profile.index.unverifiedEmails') }}</span>
+    </v-alert>
+  </BasePage>
+</template>
+
+<script>
+import { recoveryMethods } from '@/global/recoveryMethods'
+import { mfa } from '@/global/mfa'
+
+export default {
+  components: {
+  },
+  data: (vm) => ({
+    alternates: recoveryMethods.alternates,
+    mfa,
+    toggling: false,
+    hidden: vm.$user.hide === 'yes',
+    lastUpdated:
+      'last updated ' +
+      new Date(vm.$user?.password_meta?.last_changed).toLocaleDateString(navigator.language, {
+        year: 'numeric',
+        month: 'long',
+        day: 'numeric',
+      }),
+  }),
+  computed: {
+    hasUnverifiedEmails: (vm) => vm.alternates.some((m) => !m.verified),
+    items(vm) {
+      return [
+        //Todo update/add titles to locales
+        {
+          title: this.$vuetify.lang.t('$vuetify.profile.index.username'),
+          icon: 'mdi-account-circle',
+          secondary: this.$user.idp_username,
+        },
+        { divider: true, inset: true },
+        {
+          title: this.$vuetify.lang.t('$vuetify.profile.index.passwordCard.title'),
+          icon: 'mdi-account-key',
+          url: '/password/create',
+          hasPencil: true,
+          secondary: this.lastUpdated,
+        },
+        { divider: true, inset: true },
+        {
+          title: this.$vuetify.lang.t('$vuetify.profile.index.passwordRecoveryCard.title'),
+          icon: 'mdi-email-outline',
+          url: '/password/recovery',
+          hasPencil: true,
+          tooltip: 'If you lose your password we’ll send a reset link to this email address. ', //Todo add this to locales
+        },
+        { divider: true, inset: true },
+        {
+          title: this.$vuetify.lang.t('$vuetify.profile.index.manager'),
+          icon: 'mdi-account-multiple',
+          secondary: this.$user.manager_email,
+          tooltip:
+            'Your recovery contact is the person who will be sent codes to allow you to re-enter your account if all your 2-Step Verification methods are lost. It is likely your manager, and is set by HR.', //Todo add this to locales
+        },
+        { divider: true, inset: true },
+        {
+          title: this.$vuetify.lang.t('$vuetify.profile.index.dndCard.title'),
+          icon: 'mdi-security',
+          hasSwitch: true,
+          tooltip:
+            'When enabled, using the Forgot Password process will produce a message that your account does not exist. It will, however, still send you an email with a link you can use to reset your password. This is typically used by individuals in sensitive locations.', //Todo add this to locales
```

Comment:
`does not exist` could end up as `was not found` if we decide to go with the "Hide my account" wording on the UI. Again, just a suggestion, not trying to force us to go this direction.